### PR TITLE
Fix GH-1647: Add empty check before accessing values, fix phpcs

### PR DESCRIPTION
**Pull request by SID177**
Add empty check before accessing values. Fix PHPCS issues in function

Fixes https://github.com/rtCamp/rtMedia/issues/1647#issuecomment-653025519

### CODE REVIEW

File: app/main/controllers/template/rtmedia-functions.php [hunk 1]
```diff
@@ -3163,25 +3163,25 @@ function rtm_get_music_cover_art( $media_object ) {
 	$media_tags = new RTMediaTags( $file );
 	$title_info = $media_tags->title;
 	$image_info = $media_tags->image;
-	$image_mime = $image_info['mime'];
-	$mime       = explode( '/', $image_mime );
 	$id         = $media_object->id;
 
-	if ( ! empty( $image_info['data'] ) ) {
+	if ( ! empty( $image_info['data'] ) && ! empty( $image_info['mime'] ) ) {
```
Comment (ravatparmar) [inline app/main/controllers/template/rtmedia-functions.php:3168]: Ca we do the early return

File: app/main/controllers/template/rtmedia-functions.php [hunk 2]
```diff
@@ -3163,25 +3163,25 @@ function rtm_get_music_cover_art( $media_object ) {
...
-		if ( is_array( $thumb_upload_info ) && ! empty( $thumb_upload_info['url'] ) ) {
-			$media_obj->model->update(
-				array(
-					'cover_art' => $thumb_upload_info['url'],
-				),
-				array(
-					'id' => $id,
-				)
-			);
+		if ( ! empty( $mime ) && is_array( $mime ) ) {
```
Comment (ravatparmar) [inline app/main/controllers/template/rtmedia-functions.php:3171]: Here as well, can we do the early return

Comment (SID177) [reply app/main/controllers/template/rtmedia-functions.php:3171]: I don't think so, there's code after this condition

File: app/main/controllers/template/rtmedia-functions.php [hunk 3]
```diff
@@ -3163,25 +3163,25 @@ function rtm_get_music_cover_art( $media_object ) {
...
+						'cover_art' => $thumb_upload_info['url'],
+					),
+					array(
+						'id' => $id,
+					)
+				);
 
-			return $thumb_upload_info['url'];
+				return $thumb_upload_info['url'];
+			}
```
Comment (umeshnevase) [inline app/main/controllers/template/rtmedia-functions.php:3184]: Else return false

Comment (SID177) [reply app/main/controllers/template/rtmedia-functions.php:3184]: There's a code which is getting executed after this block. So I think returning false in else isn't a good idea.

**Pull request merged**